### DISCUSSION TITLE
Typo3 as Gatsby Backend

### DISCUSSION THREAD

**Discussion by jannis-diggi** [gatsbyjs/gatsby#37985] [Ideas / Feature Requests]
Me and my colleagues are looking for headless Typo3 uses. 
Since others in our company were already using Gatsby, we thought Typo3 might be an applicable Backend for Gatsby. 
We have looked and found no plugin regarding Typo3 (not even a discussion in this repository). 
Thus we want to ask, is there something we didn't see: Is a plugin already there (or in development), or a request for such kind?

**Comment by LekoArts** [comment]
Hi!

I also couldn't find any Gatsby source plugin for Typo3. We just released a new tutorial on how to create a source plugin: https://www.gatsbyjs.com/docs/tutorial/creating-a-source-plugin/

I'd assume you have to use https://extensions.typo3.org/extension/headless to be able to use Typo3 headless. Be the change you want to see in the world and create a source plugin :)

We (Gatsby) only maintain the plugins in this repository and won't add any additional ones.
